drivers/Makefile

```diff
@@ -92,6 +92,8 @@ OBJS = $(AOBJS) $(COBJS)
 
 BIN = libdrivers$(LIBEXT)
 
+CFLAGS += ${INCDIR_PREFIX}$(TOPDIR)$(DELIM)sched
```

Comment:
should we move this_cpu to public include/nuttx/sched/sched.h?

Reply:
Done